trunk/SUAVE/Methods/Weights/Buildups/Common/prop.py

```diff
@@ -0,0 +1,312 @@
+## @ingroup Methods-Weights-Buildups-Common
+
+# prop.py
+#
+# Created: Jun, 2017, J. Smart
+# Modified: Feb, 2018, J. Smart
+
+#-------------------------------------------------------------------------------
+# Imports
+#-------------------------------------------------------------------------------
+
+from SUAVE.Core import Units
+from SUAVE.Attributes.Solids import (
+    bidirectional_carbon_fiber, honeycomb, paint, unidirectional_carbon_fiber, acrylic, steel, aluminum, epoxy, nickel, rib)
+import numpy as np
+import copy as cp
+
+#-------------------------------------------------------------------------------
+# Prop
+#-------------------------------------------------------------------------------
+
+def prop(prop,
+         maximum_thrust,
```

Comment:
a lot of this data can be gathered from the propeller: Number of blades and some ratios.

Reply:
Number of blades changed to reference propeller data. User can override defaults to use propeller data if desired, but since the method depends on an assumed and approximate geometry for the prop, I prefer to leave them with default values to remain workable.